pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -176,6 +180,9 @@ def _check_inventory(self, lookup_ids=[]):
                             pokemon.get('individual_stamina', 0)
                         ]})
                     matched_pokemon.append(pokemon)
+                    #remove as egg and add as pokemon
+                    inventory.pokemons().remove(pokemon['id'])
+                    inventory.pokemons().add(inventory.Pokemon(pokemon))
```

Comment:
I don't think so because this function also fetch the km walked on each eggs to know if they have to be hatched. So right now the egg incubator does not work if it does not query the api at every loop.
Ideally, walkers would update km walked somewhere and egg incubator would use that to know when to hatch eggs, or at least when to query api to check. That would require complete rewrite of this task.
What is the phone app doing anyway ? Is it querying the inventory regularly ?